RELEASE_PROCESS.md

```diff
@@ -1,73 +1,61 @@
 # Our Release Process
 
-The release process for BigchainDB server differs slightly depending on whether it's a minor or a patch release.
+## Notes
 
-BigchainDB follows 
+BigchainDB follows
 [the Python form of Semantic Versioning](https://packaging.python.org/tutorials/distributing-packages/#choosing-a-versioning-scheme)
 (i.e. MAJOR.MINOR.PATCH),
 which is almost identical
-to [regular semantic versioning](http://semver.org/)
-except release candidates are labelled like
-`3.4.5rc2` not `3.4.5-rc2` (with no hyphen).
+to [regular semantic versioning](http://semver.org/), but there's no hyphen, e.g.
 
+- `0.9.0` for a typical final release
+- `4.5.2a1` not `4.5.2-a1` for the first Alpha release
+- `3.4.5rc2` not `3.4.5-rc2` for Release Candidate 2
 
-## Minor release
+We use `0.9` and `0.9.0` as example version and short-version values below. You should replace those with the correct values for your new version.
 
-A minor release is preceeded by a feature freeze and created from the 'master' branch. This is a summary of the steps we go through to release a new minor version of BigchainDB Server.
+We follow [C4, the Collective Code Construction Contract](https://github.com/bigchaindb/BEPs/tree/master/1), so a release is just a [tagged commit](https://git-scm.com/book/en/v2/Git-Basics-Tagging) on the `master` branch, i.e. a label for a particular Git commit.
```

Comment:
Now that I just noticed, should we say *We follow BEP-1..." because it's our own version of C4?

Reply:
ok